OrcScala/src/orc/API.scala

```diff
@@ -24,6 +24,12 @@ import orc.progress.ProgressMonitor
 import orc.values.Signal
 import scala.util.parsing.input.Position
 import orc.compile.CompilerFlagValue
+import orc.error.runtime.HaltException
+import orc.values.Field
+import orc.error.runtime.{ TypeNoSuchMemberException, UncallableTypeException, NoSuchMemberException, UncallableValueException, TypeDoesNotHaveMembersException }
+import orc.error.runtime.DoesNotHaveMembersException
+
+// TODO: This file is huge and covers many unrelated things. Split into multiple sub API with limited relationships. Also it's not clear where to put utility implementations/bases for these APIs.
```

Comment:
I was thinking that since OrcRuntime depends on Invoker, Accessor, and relatives, all of it can go into a OrcRuntimeInterface.scala file, and then the canned implementations go into InvocationBehaviorUtilities.scala.